2nd chunk needed for:
Error: could not find libjava.so
Error: Could not find Java SE Runtime Environment.
--- launcher/launchers.in.orig	Wed Feb  3 14:47:43 2016
+++ launcher/launchers.in	Wed Mar  9 17:35:04 2016
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!${TRUEPREFIX}/bin/bash
 
 JAVA=@JAVA@
 LAUNCHER_BOOTCLASSPATH=@LAUNCHER_BOOTCLASSPATH@
@@ -95,6 +95,6 @@ while [ "$j" -lt "${#ARGS[@]}" ]; do
   k=$((k+1))
 done
 
-exec -a "$PROGRAM_NAME" "${COMMAND[@]}"
+exec "${COMMAND[@]}"
 
 exit $?
